src/msgAggregatorWorker.ts

```diff
@@ -31,10 +33,10 @@ export const parseSerialMessages = (
   // so we need to discard it and start aggregating from the first encountered separator
   let joinMessages = messages.join("");
   if (discardFirstLine) {
-    const firstSeparatorIndex = joinMessages.indexOf(separator);
-    if (firstSeparatorIndex > -1) {
+    const separatorMatch = separatorRegex.exec(joinMessages);
+	  if (separatorMatch && separatorMatch.index > -1) {
```

Comment:
```suggestion
    if (separatorMatch && separatorMatch.index > -1) {
```
Fix indentation (this project uses spaces, not tabs)